Makefile

```diff
@@ -48,45 +44,11 @@ distclean:
 	git clean -fdx
 	git submodule foreach git clean -fdx
 
-venv:
-	@echo "** setting up virtualenv **"
-	mkdir -p var/cache/pip
-	virtualenv env
-	./env/bin/pip install --download-cache var/cache/pip $(OL_VENDOR)/openlibrary.pybundle
-
-install_solr:
-	@echo "** installing solr **"
-	mkdir -p var/lib/solr var/cache usr/local
-	wget -c $(OL_VENDOR)/$(SOLR_VERSION).tgz -O var/cache/$(SOLR_VERSION).tgz
-	cd usr/local && tar xzf ../../var/cache/$(SOLR_VERSION).tgz && ln -fs $(SOLR_VERSION) solr
-
-setup_coverstore:
-	@echo "** setting up coverstore **"
-	$(PYTHON) scripts/setup_dev_instance.py --setup-coverstore
```

Comment:
This seems like a good command to document somewhere (e.g. wiki) as we haven't had to stand up a coverstore service for dev or production (and I imagine we soon may)
@cdrini

Reply:
@anandology deprecated in 2015, we're safe to rm reference.
https://github.com/internetarchive/openlibrary/commit/6c00c6d7084602991cfcd3aa854b350e6ab80dd5#diff-4d3a94d17591ed9f3d442836fb4f9e4c

Thank you @cdrini for this archaeology